From: Debian Science Maintainers
 <debian-science-maintainers@lists.alioth.debian.org>
Date: Tue, 24 Nov 2015 16:46:14 +0100
Subject: copyright-date-typo-fix

Forwarded: no
#but it should be
Last-Update: 2015-11-20
---
 sensor_msgs/src/sensor_msgs/point_cloud2.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sensor_msgs/src/sensor_msgs/point_cloud2.py b/sensor_msgs/src/sensor_msgs/point_cloud2.py
index 9bfc0c8..b19f103 100644
--- a/sensor_msgs/src/sensor_msgs/point_cloud2.py
+++ b/sensor_msgs/src/sensor_msgs/point_cloud2.py
@@ -2,7 +2,7 @@
 
 # Software License Agreement (BSD License)
 #
-# Copyright (c) 208, Willow Garage, Inc.
+# Copyright (c) 2008, Willow Garage, Inc.
 # All rights reserved.
 #
 # Redistribution and use in source and binary forms, with or without
